server/parsec/components/user.py

```diff
@@ -698,3 +709,20 @@ async def api_tos_accept(
                 return tos_cmds.latest.tos_accept.RepNoTos()
             case UserAcceptTosBadOutcome.TOS_MISMATCH:
                 return tos_cmds.latest.tos_accept.RepTosMismatch()
+
+    @api
+    async def api_list_frozen_users(
+        self,
+        client_ctx: AuthenticatedClientContext,
+        req: authenticated_cmds.latest.list_frozen_users.Req,
+    ) -> authenticated_cmds.latest.list_frozen_users.Rep:
+        outcome = await self.list_frozen_users(client_ctx.organization_id, client_ctx.user_id)
+        match outcome:
```

Comment:
You should also add a `case UserListFrozenUsersBadOutcome.AUTHOR_REVOKED:` 

Note the test `server/tests/api_v4/authenticated/test_list_frozen_users.py::test_authenticated_list_frozen_users_http_common_errors[author_revoked]` has passed even without this: this is because the authentication system is triggered first and handle the case.

So this is mostly to stay consistent with the rest of the APIs (we have plan in the long run to remove the organization/user exists/not revoked/not expired/not frozen check from all the APIs in order have this totally factorized, but now is not the time ^^)